<launch>
    <node pkg="rviz"  type="rviz"  name="rviz"  args="-d $(find urban_nav)/config/display.rviz"></node>

    <param name="robot_description" textfile="$(find urban_nav)/model/car_model/default.urdf"/>
    <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"/>
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher"/>
    <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster"
          args="0 0 0 0 0 0 world map 100"/>
          
    <node pkg="urban_nav"  type="odom_node"  name="odom_node"  output="screen"></node>
</launch>